README.md

```diff
@@ -15,22 +15,36 @@ We have nightly and versioned Docker images, Debian packages, RPM packages, and
 
 This project requires Go 1.13 and Go module support.
```

Comment:
While you're in here:
```suggestion
This project requires Go 1.15 and Go module support.
```

Reply:
And a suggestion, it could be nice to suggest using something like [GVM](https://github.com/moovweb/gvm) to manage Go versions. I saw some members of the cloud team talking about it when I started, and it's been nice for juggling versions across projects.

Reply:
We actually do suggest that here:

https://github.com/influxdata/influxdb/blob/master/CONTRIBUTING.md#building-testing-and-installing 

Looks like these two sections need to be collapsed - the one in `CONTRIBUTING.md` looks like it is still for 1.x